python/setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="aswfdocker",
+    version="0.1.0",
+    author="Aloys Baillet",
+    author_email="aloys.baillet+github@gmail.com",
+    description="ASWF Docker Utilities",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/AcademySoftwareFoundation/aswf-docker",
```

Comment:
Hard coded ASWF github path

Reply:
the `setup.py` file needs to live on its own and not import files as import failures would happen during install... so these will have to stay duplicated...